content/en/blog/_posts/2023-09-13-userns-stateful-pods/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: blog
+title: "User namespaces: Now alpha support to run stateful pods too!"
```

Comment:
grammer nit: `User namespaces: now supports running stateful pods in alpha!` or something

Reply:
Thanks!